### PR TITLE
1.6.0 Modification for mixer-layout

### PR DESCRIPTION
**Pull request by Energizz**

### CODE REVIEW

**Review by f-ld — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,141 +1,141 @@
...
-    The Platform triggers either the `joinConference` button or the `replayConference` button, depending on whether the conference is live or a replay.
-    The mixer layout then uses the Dolby Interactivity APIs Web SDK to join the conference as a mixer and creates the layout based on the number and state of the participants in the conference.
-
+   The input fields are used to define the details of the conference to be used to produce the presentation.
+   When one of the buttons is clicked, the layout app takes the values in these fields to connect to the conference or the recording.
+
+   These inputs and buttons are required and in this app they are found in the file `src/app/components/ConferenceRoom.js`.
+
+   ```html
+   <input
```
Comment (FabienLavocat) [inline README.md]: I would use a one-liner here to keep consistency with the other input elements

File: README.md [hunk 2]
```diff
@@ -1,141 +1,141 @@
...
+   />
+   <input type="hidden" value="catToken" id="catToken" name="catToken" />
+   <input type="hidden" value="voxeet" id="conferenceId" name="conferenceId" />
+   <input type="hidden" value="refreshUrl" id="refreshUrl" name="refreshUrl" />
+   <input type="hidden" value="1234" id="thirdPartyId" name="thirdPartyId" />
+   <input type="hidden" value="stream" id="layoutType" name="layoutType" />
+   <button id="joinConference" onClick="{this.launchConference.bind(this)}">
+     Join conference
+   </button>
+   <button
```
Comment (FabienLavocat) [inline README.md]: I would use a one-liner here to keep consistency with the other button above.

File: README.md [hunk 3]
```diff
@@ -1,141 +1,141 @@
...
+   These inputs and buttons are required and in this app they are found in the file `src/app/components/ConferenceRoom.js`.
+
+   ```html
+   <input
+     type="hidden"
+     value="accessToken"
+     id="accessToken"
+     name="accessToken"
+   />
+   <input
```
Comment (FabienLavocat) [inline README.md]: I would use a one-liner here to keep consistency with the other input elements

File: package.json [hunk 4]
```diff
@@ -35,12 +36,8 @@
...
     "webpack-dev-server": "^2.8.2"
   },
-  "peerDependencies": {
-    "@voxeet/voxeet-web-sdk": "^2.3.1",
-    "express": "^4.0.0"
-  },
   "scripts": {
     "start": "webpack-dev-server -d",
-    "build": "npx webpack --output-filename out.js --output-path dist"
+    "build": "npx webpack  --output-filename out.js --output-path dist"
```
Comment (FabienLavocat) [inline package.json]: Extra space added

File: src/app/components/ConferenceRoom.js [hunk 5]
```diff
@@ -211,10 +252,6 @@ class ConferenceRoom extends Component {
     const { isLaunch, layoutType } = this.state;
     let count = -1;
     let participantConnected = participants.filter((p) => p.isConnected);
-    let className = "tiles-list";
```
Comment (FabienLavocat) [inline src/app/components/ConferenceRoom.js:214]: the line 262 in this file need to be updated from:
```
{(consumerKey == null || consumerKey == null) && (
```
to:
```
{(consumerKey == null || consumerSecret == null) && (
```

File: src/app/components/ConferenceRoom.js [hunk 6]
```diff
@@ -211,10 +252,6 @@ class ConferenceRoom extends Component {
     const { isLaunch, layoutType } = this.state;
```
Comment (FabienLavocat) [inline src/app/components/ConferenceRoom.js]: `layoutType` is never used

File: src/app/components/ConferenceRoom.js [hunk 7]
```diff
@@ -239,7 +276,7 @@ class ConferenceRoom extends Component {
               <div>
                 <h3>Live conference</h3>
                 <input
-                  placeholder="Conference name (alias)"
+                  placeholder="Conference name"
```
Comment (FabienLavocat) [inline src/app/components/ConferenceRoom.js]: Maybe `Conference alias`?

File: src/app/components/ConferenceRoom.js [hunk 8]
```diff
@@ -211,10 +252,6 @@ class ConferenceRoom extends Component {
     const { isLaunch, layoutType } = this.state;
     let count = -1;
     let participantConnected = participants.filter((p) => p.isConnected);
-    let className = "tiles-list";
-    if (participantConnected.length <= 15) {
-      className += "-" + participantConnected.length;
-    }
     return (
```
Comment (FabienLavocat) [inline src/app/components/ConferenceRoom.js:255]: Could we change the title of this page (in the h1 tag) to `Dolby Mixer Layout Testing`

File: src/app/actions/ConferenceActions.js [hunk 9]
```diff
@@ -83,27 +83,18 @@ export class Actions {
               return VoxeetSDK.conference
                 .join(conference, {
                   constraints,
+                  conferenceAccessToken:
+                    catToken && catToken.length > 0 ? catToken : null,
                   mixing: {
                     enabled: true,
                   },
                   userParams,
                   audio3D: false,
```
Comment (FabienLavocat) [inline src/app/actions/ConferenceActions.js]: Should we remove that `audio3D`?

**Review by FabienLavocat — CHANGES_REQUESTED** [review]

File: src/app/components/ConferenceRoom.js (hunk 8 above)
Comment (Energizz) [reply src/app/components/ConferenceRoom.js:255]: Of course :)

**Review by FabienLavocat — APPROVED** [review]
Thanks :)

**Pull request merged**